packages/graphql/src/translate/projection/subquery/translate-cypher-directive-projection.ts

```diff
@@ -230,6 +243,37 @@ function createCypherDirectiveApocProcedure({
     return apocClause;
 }
 
+function createExperimentalCypherStatement({
```

Comment:
I know we're marking this as experimental for our users, but is it really experimental for us? Not worth naming this function with a more long-term name?